src/video_core/renderer_vulkan/vk_graphics_pipeline.h

```diff
@@ -36,11 +36,14 @@ struct GraphicsPipelineKey {
     vk::Format depth_format;
     vk::Format stencil_format;
 
-    Liverpool::DepthControl depth_stencil;
-    u32 depth_bias_enable;
+    bool depth_test_enable;
```

Comment:
These bools can be made into a bitfield struct to reduce the used size

Reply:
Sure, done